src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.ts

```diff
@@ -97,7 +103,9 @@ export class EditTopicDialogComponent {
     const manualTopicOption: TopicOption = {
       type: this.TopicSelectionEnum.MANUAL,
       label: 'Manual',
-      description: `Entered manually by the user${this.isGitHubAppEntry ? 'in the .dockstore.yml.' : '.'}`,
+      description: `Entered manually by the user.${
+        this.isGitHubAppEntry ? ` Edit the topic field in the .dockstore.yml file on GitHub.` : ''
```

Comment:
Nit, I find this a little confusing... First it tells you that a user (another user?) entered the value manually. Then, if it's a dockstore.yml based entry, it tells you (not the user from the first sentence) to edit the topic field in the .dockstore.yml.

How about something like, roughly, needs polishing, really not wild about the second item:

* Specified in the .dockstore.yml. Update the .dockstore.yml to change the topic.
* Specify the topic sentence here

Reply:
Updated screenshot:

![Screenshot 2024-09-05 at 13-37-12 Dockstore My Notebooks](https://github.com/user-attachments/assets/4b8c60f8-3254-4d12-a087-e0d8ffe73808)